### ISSUE TITLE
Bad attachement

### ISSUE THREAD

**Issue by AqwelF** [jensstein/oandbackup#215]
Hi,

I have aleays error when i save my apps+data : bad attachement for all apps.
See error message.

Busybox is ok.

Phone : honor 9
Os : 8.1 AOSP FLOSS
App version : 0.2.13

![](https://i.imgur.com/4sIVfOU.png)

**Comment by ildar** [comment]
You seem to have bad `BusyBox` settings, please check

**Comment by ildar** [comment]
I mean bad settings in oandbackup.
You can Clear Data for the oandbackup app and start again.
If very troubled please contact me privately
-- 
Ildar Mulyukov,
(ΙΧΘΥΣ) child of God

email: ildar.mulyukov@gmail.com
matrix: @ildar:matrix.org
GoogleTalk: ildar.mulyukov@gmail.com
blog: http://johan-notes.blogspot.com/

**Comment by AqwelF** [comment]
I make a fresh install, without touching anythin look better i have just few errors.

Before i just change directory for backup to my SD card, do you think this can occured the trouble i had ?

Now i have this 
![](https://i.imgur.com/s7Soa3q.png)

**Comment by acuna-personal** [comment]
Stupid question: does your phone is rooted?

**Comment by AqwelF** [comment]
Yes it is :) the backup work fine for like 110 apps. And sux just with those

**Comment by acuna-personal** [comment]
Ahah, okay) As about your screenshots as I see reading it by its strings:

1) `wpa_suplicant.conf` file doesn't exists, seems Android store Wi-Fi access points at another place, changes to OAB is requred, so now you can't backup your Wi-Fi Access points at this time.
2) Invalid argument for `cp` means that filenames have a denied symbols, `cp` Linux command which is used by OAB can't handle the `:` and folders, started with dot, so unfortunately you can't backup some system apps like E-Mail, Shazam and other apps which have a `.Fabric` folder and something like it.

**Comment by AqwelF** [comment]
Ok
1/ what can i do for my wifi backup ?
2/ now onyl xda app doesnt backup. Not a problem if only one app dont..

**Comment by acuna-personal** [comment]
1. You can check if `/data/misc/wifi/wpa_suplicant.conf` file is exists and create new PR request to the developer if it not (but I think he read this theme and can see your problem. This project is not in so active development but sometimes it have an updates ;) ).
2. I'm so sorry, but can you formulate your question more clearly if it's question?) I can't understand it)

**Comment by AqwelF** [comment]
Thank you ! #2 is not a qiestion ;)

**Comment by acuna-personal** [comment]
You wellcome ;) But what you mean after all? I see that you can't backup not only XDA app.

**Comment by AqwelF** [comment]
Backup problems from my first post are gone when i put back saving folder into intern storage. (I switch to SD before and make oandbackup buggy).

Now just xda and wifi errors still appears.

**Comment by acuna-personal** [comment]
@AqwelF wow, thanks, seems that there's a way to make a `cp` working.

**Comment by acuna-personal** [comment]
@AqwelF wow, thanks, seems that there's a way to make a `cp` command working correctly.
